.github/workflows/ci.yml

```diff
@@ -537,6 +537,27 @@ jobs:
     - name: run tests
       run: ./ci/do_ci.sh bazel.test
 
+  bazel_no_bzlmod_test:
+    name: Bazel
```

Comment:
There is already a CI build named Bazel, please pick something more descriptive.